### ISSUE TITLE
[ProgressView] Internal issue: b/141681184 (MDCProgressView fails accessibility tests)

### ISSUE THREAD

**Issue by material-mirror** [material-components/material-components-ios#8509]
This was filed as an internal issue. If you are a Googler, please visit [b/141681184](http://b/141681184) for more details.

---
#### Internal data
- Associated internal bug: [b/141681184](http://b/141681184)
